paper-badge.html

```diff
@@ -28,7 +34,7 @@
 
     <div>
       <paper-button id="btn">Status</paper-button>
-      <paper-badge for="btn" label="♥︎"></paper-badge>
+      <paper-badge icon-badge for="btn" label="favorite"></paper-badge>
```

Comment:
Yeah, I think that would be best. Generally with images and icons we don't infer things from the source or the icon name, we make the user specify it themselves. I would document this really well though :)

/cc @cdata.